.travis.yml

```diff
@@ -7,27 +6,28 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27  
```

Comment:
@melund as you can see conda-smithy saw the `skip: True  # [py2k]` and removed the Python 2 builds for you.

PS: So xonsh is Python 3 only now?